packages/deepbook/sources/state/state.move

```diff
@@ -293,6 +293,7 @@ module deepbook::state {
 
             let base_volume = fill.base_quantity();
             let quote_volume = fill.quote_quantity();
+            self.history.add_volume(base_volume, account.active_stake());
```

Comment:
^

Reply:
We should only be adding if the fill is not expired right?

Reply:
this should go to line 305 https://github.com/MystenLabs/deepbookv3/pull/186/files#diff-43b392ff4a2456a7ba3301644215f06f0150c50094e1da5ca92ab40b54e810a8R305